bindings/pyroot_experimental/PyROOT/python/ROOT/pythonization/_rtensor.py

```diff
@@ -0,0 +1,110 @@
+# Author: Stefan Wunsch CERN  07/2019
+
+################################################################################
+# Copyright (C) 1995-2019, Rene Brun and Fons Rademakers.                      #
+# All rights reserved.                                                         #
+#                                                                              #
+# For the licensing terms see $ROOTSYS/LICENSE.                                #
+# For the list of contributors see $ROOTSYS/README/CREDITS.                    #
+################################################################################
+
+from ROOT import pythonization
+from libROOTPython import GetEndianess, GetTensorDataPointer, GetSizeOfType, AsRTensor
+from ROOT.pythonization._rvec import _array_interface_dtype_map
+import cppyy
+
+
+def get_array_interface(self):
+    cppname = type(self).__cppname__
+    for dtype in _array_interface_dtype_map:
+        if not cppname.find("RTensor<{},".format(dtype)) is -1:
```

Comment:
Didn't you check this already in `add_array_interface_property`? Instead of iterating, you could just parse `dtype` from `cppname` right?

Reply:
Hm I wanted to avoid a regex call. But I can go with a rather simple string manipulation by cutting away everything besides the data type.

Reply:
Done